app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from os import mkdir, replace
+import os
```

Comment:
You are importing `mkdir` and `replace` from `os` in line 1 and then importing the entire `os` module in line 2. This is redundant. You can either import specific functions or the entire module, but not both. Consider removing the specific imports in line 1 if you are using the full module import in line 2.